Extensions/find_inactives.js

```diff
@@ -336,7 +336,7 @@ XKit.extensions.find_inactives = new Object({
 	},
 	trigger_unfollow: function() {
 		var username = add_tag;
-		var removalTarget = jQuery('.find-inactives-blog[data-url="http://' + username + '.tumblr.com/');
+		var removalTarget = jQuery('.find-inactives-blog[data-url="http://' + username + '.tumblr.com/"]');
```

Comment:
this code would be cleaner to read and easier to diagnose if it was using template strings.

Reply:
What's the lowest version of Safari that we're willing to support? Seems like 9.1 and above have support for template strings.

Reply:
Update: it would seem Safari 9 is no longer considered supported.